lib/vtt.js

```diff
@@ -1469,4 +1478,4 @@
 
   global.WebVTT = WebVTT;
 
-}(this));
+}(this, (this.vttjs || {})));
```

Comment:
And in that case, we could make the options object something like:
```js
new WebVTT.Parser({
  window: window,
  decoder: WebVTT.StringDecoder,
  useShims: true
});
```
and it'll just use it's own reference.
Though, now that I think about it, we probably could do this type of API with what we have now, unless you think it's a good idea to be able to have the user pass in the exact shim that should be used.